### PR TITLE
Reset pod installation backoff time if new manifest is received.

### PR DESCRIPTION
**Pull request by mpuncel**
Commit f98b0a1116cc82caab27a6202d91aea4d0b1019f introduced an
exponential backoff for pod installation and launching when an error
occurs. The backoff starts at 1 second and doubles on each error. The
backoff time is only reset when a pod launch is successful.

This commit also resets the backoff time if a new manifest is received,
which will reduce the amount of time it requires for a "fixed" pod
manifest update to be processed. For example, if a manifest is deployed
with a bad launchable URL and the backoff time rises to a minute, it is
desirable that a corrected manifest can be scheduled and processed
immediately, instead of after a large backoff time.

### CODE REVIEW

File: pkg/preparer/orchestrate.go [hunk 1]
```diff
@@ -186,6 +186,7 @@ func (p *Preparer) handlePods(podChan <-chan ManifestPair, quit <-chan struct{})
 		case <-quit:
 			return
 		case nextLaunch = <-podChan:
+			backoffTime = 1 * time.Second
```
Comment (mpuncel) [inline pkg/preparer/orchestrate.go]: no, does it need to be?

Comment (mpuncel) [inline pkg/preparer/orchestrate.go]: historically this was hardcoded inside a `time.After()`

Comment (petertseng) [inline pkg/preparer/orchestrate.go]: maybe at least name something `minimumBackoffTime := 1 * time.Second`?

**Comment by petertseng** [comment]
Is this important enough that I should not roll preparers to production without this?

**Comment by mpuncel** [comment]
@petertseng I think it's okay to proceed, this just means updates will be delayed by a minute iff the previous version never installed or launched correctly

**Pull request merged**